{"ms.devlang":"csharp","author":"erickson-doug","ms.author":"douge","manager":"douge","breadcrumb_path":"breadcrumb/toc.json","ms.topic":"reference","ms.service":"service-fabric","ms.date":"11/16/2016","search.ms_sitename":"Docs","search.ms_docsetname":"azuredotnet","version":0,"locale":"en-us","site_name":"Docs","search.ms_product":"Azure","depot_name":"Azure.azuredotnet","ref_skeleton_gitcommit":"https://github.com/Azure/azure-docs-sdk-dotnet/blob/8ece26063f2f27414f0bad873159ee6212f3448d/docs-ref-manual/Data/ServiceFabric/Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.yml","original_ref_skeleton_git_url":"https://github.com/Azure/azure-docs-sdk-dotnet/blob/live/docs-ref-manual/Data/ServiceFabric/Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.yml","open_to_public_contributors":false,"api_name":["Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.Data","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.get_Data","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation..ctor","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.FromException","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.ToException"],"api_location":["Microsoft.ServiceFabric.Services.dll"],"topic_type":["apiref"],"api_type":["Assembly"],"f1_keywords":["Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation","Microsoft::ServiceFabric::Services::Communication::RemoteExceptionInformation","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.Data","Microsoft::ServiceFabric::Services::Communication::RemoteExceptionInformation::Data","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.#ctor","Microsoft::ServiceFabric::Services::Communication::RemoteExceptionInformation::#ctor","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.FromException","Microsoft::ServiceFabric::Services::Communication::RemoteExceptionInformation::FromException","Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation.ToException","Microsoft::ServiceFabric::Services::Communication::RemoteExceptionInformation::ToException"],"dev_langs":["csharp","vb"],"updated_at":"2016-12-26 05:25 AM","document_id":"9bf30b74-f6b8-c888-59df-48118a141cb7","layout":"Reference","pagetype":"Reference","title":"RemoteExceptionInformation class | Microsoft Docs","description":"Fault type used by Service Remoting to transfer the exception details from the Service Replica to the client.\n","toc_asset_id":"toc.json","toc_rel":"toc.json","content_git_url":"","source_url":"https://msazure.visualstudio.com/DefaultCollection/One/_git/WindowsFabric?path=src/prod/src/managed/Microsoft.ServiceFabric.Services/Communication/RemoteExceptionInformation.cs&version=GBdevelop&line=14","ms.assetid":"Microsoft.ServiceFabric.Services.Communication.RemoteExceptionInformation","canonical_url":"https://ppe.docs.microsoft.com/en-us/dotnet/api/microsoft.servicefabric.services.communication.remoteexceptioninformation","is_dynamic_rendering":true,"content_type":"application/json","publish_version":"{\"opbuild.templates.docs\":\"1.30.6\",\"commit_id\":\"5309f248950863ba95719cb7f61643b812266628\",\"branch\":\"live\",\"opbuild.scripts\":\"1.30.6\",\"opbuild.htmltopdftool\":\"1.30.6\",\"opbuild.gitcontributorinformationresolver\":\"1.30.6\",\"opbuild_service\":\"1.30.130.0\",\"docfx.console\":\"2.15.3\",\"opbuild.resolvedependencyconsole\":\"1.30.6\",\"opbuild.schemavalidationconsole\":\"1.30.6\",\"opbuild.redirectionresolver\":\"1.30.6\",\"opbuild.templates.intellisense\":\"1.30.6\",\"dependencies\":[{\"branch\":\"master\",\"path\":\"_themes\",\"url\":\"https://github.com/Microsoft/templates.docs.msft\",\"commit_id\":\"f5f2d9e7e224c9cb502966264b96d2abe29a1f29\",\"opbuild.templates.common\":\"1.30.6\"}]}"}